Sources/Model/ObjC/Common/TracksDeviceInformation.m

```diff
@@ -21,8 +21,10 @@ @interface TracksDeviceInformation ()
 @property (nonatomic, assign) BOOL isReachableByWiFi;
 @property (nonatomic, assign) BOOL isReachableByWWAN;
 
+#if TARGET_OS_SIMULATOR
```

Comment:
Should this be `TARGET_OS_IPHONE` instead? Otherwise, these two properties won't be included when building against real device, right?